js/zb.js

```diff
@@ -334,37 +334,115 @@ module.exports = class zb extends Exchange {
         //         },
         //     }
         //
-        const keys = Object.keys (markets);
+        const contracts = await this.contractV2PublicGetConfigMarketList (params);
+        //
+        //     {
+        //         BTC_USDT: {
+        //             symbol: 'BTC_USDT',
+        //             buyerCurrencyId: '6',
+        //             contractType: '1',
```

Comment:
I think you're right about this but contractType can only return 1 currently since their is only swaps available on ZB and it doesn't have any information about different contract types yet